share/smack/include/pthread.h

```diff
@@ -0,0 +1,71 @@
+//
```

Comment:
Question: why are we providing header files at all?

I would imagine that the key use case here would be that an application uses `pthreads` and that we want to provide our false implementation of `pthreads` in place of the real one. In that case, there ought to be no change to the existing `pthreads` header files. The only change would be against which `pthreads` implementation we link. Right?

I think it would be best not to include these header files in SMACK.